p2p_wallet/Extensions/Wallet+Extensions.swift

```diff
@@ -12,8 +12,12 @@ extension Array where Element == Wallet {
     var totalAmountInCurrentFiat: Double {
         reduce(0) { $0 + $1.amountInCurrentFiat }
     }
-    
+
+    var totalAmount: Double? {
```

Comment:
Bad code. There are difference tokens and we can not performance additional operator.

Reply:
what do you mean? that's just for the empty state. do you think I should hide this param to "isTotalAmountEmpty" or just perform this count in HomeViewModel?

Reply:
I guess second one is better